frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -783,9 +767,11 @@ export default ({
     async initializeState (forceClearMessages = false) {
       // NOTE: this state is rendered using the chatroom contract functions
       //       so should be CAREFUL of updating the fields
-      const chatroomID = this.nonReactive.renderingChatRoomId
       const messageState = await this.generateNewChatRoomState(forceClearMessages)
-      if (!this.checkEventSourceConsistency(chatroomID)) return
+
+      // If user has since switched to another chatroom, no need to update 'this.messageState'
+      if (this.summary.chatRoomID !== this.ephemeral.renderingChatRoomId) return
```

Comment:
For consistency's sake, should this use `this.chatroomHasSwitchedFrom` too? (check the rest of the code for similar inconsistencies)